### ISSUE TITLE
Mouf instance creation errors

### ISSUE THREAD

**Issue by snowwhite686** [thecodingmachine/mouf#136]
Hi,

i am a newbie to mouf. I could install the mouf using composers and its latest version (both the composer and mouf). Now when i navigate to included file analysis, it says there are few errors like the screenshot.
![capture_mouf](https://cloud.githubusercontent.com/assets/3541070/20827842/52da7478-b89a-11e6-8e98-4d4fabf2bcb6.JPG)

I tried composer update, deleted vendor folder and again install and now am stuck with this. Can you help further.?

my local url is like http://localhost/gparant/vendor/mouf/mouf/

I have attached my local installation zip copy, please help.

[https://drive.google.com/file/d/0B4hLiOu9a0MIanhDOVcyeEFDZ2M/view?usp=sharing](url)
thanks

**Comment by moufmouf** [comment]
Hi @snowwhite686 ,

If I understand correctly, these errors are displayed on the "included file analysis" page.

You do not need to worry about those errors. Mouf will try to analyse all classes (including all classes in the `vendor` directory). Some of those classes, Mouf cannot import. This is not a big deal and not a problem in your case.

This screen is still useful, because if you see some of the classe of **your** project there (the ones that are not in the vendor directory), then Mouf will display an appropriate error message for those.

To sum it up, if you see this screen, it is likely you have a working copy of Mouf. Congratulations :)

**Comment by snowwhite686** [comment]
ya, but as per the sample tutorial, i should see a warrior class while clicking on the listing buttton, its not there? could you please tell me why is this missing and how can i get it back?

**Comment by moufmouf** [comment]
Ping @gparant

**Comment by snowwhite686** [comment]
Nope, i think you didn't get my point. 

i am talking about the documentation video sample from mouf website. There a sample dependency is shown with hero instance and class selected as new warrior. In my installation copy this class is not showing up for me to select further.

I hope its clear?

http://mouf-php.com/packages/mouf/mouf/doc/mouf_di_ui.md

**Comment by moufmouf** [comment]
Hey @snowwhite686 

These classes are located in a "sample repository" (according to the documentation, it is: https://github.com/thecodingmachine/samples.tutorial.git )

Did you start from this repository? You need to have the "Warrior" and "Wizard" classes in your code to see them in Mouf.

**Comment by snowwhite686** [comment]
yes, i see it. By the way, i tried to use composer require mouf/database.tdbm but it says dbal wrapper ~ is also needed. If i include that , again connected missing.

Do you have the list of db commands for composer for mouf?

**Comment by moufmouf** [comment]
Weird.... could you post the exact error message you are getting for TDBM install?

Did you follow instructions here: http://mouf-php.com/packages/mouf/database.tdbm/doc/install.md ?

**Comment by snowwhite686** [comment]
E:\xampp\htdocs\gparant>composer require mouf/database.tdbm
Using version ^4.2 for mouf/database.tdbm
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for mouf/database.tdbm ^4.2 -> satisfiable by mouf/database.tdbm[v4.2.0].
    - mouf/database.tdbm v4.2.0 requires mouf/database.doctrine-dbal-wrapper ~1.1 -> satisfiable by mouf/database.doctrine-dbal-wrapper[1.1.x-dev] but these conflict with your requirements or minimum-stability.


Installation failed, reverting ./composer.json to its original content.

yes i followed the same link only..

**Comment by moufmouf** [comment]
Ok, I see.

You need to add those lines in your composer.json file at the root level:

```
    "minimum-stability": "dev",
    "prefer-stable": true,
```
